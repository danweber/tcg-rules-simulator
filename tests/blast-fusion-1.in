VERSION:2
FIELD1:Blue,CS2-AnimeSword,CS2-01 Yellow1,CS1-Obsidian,CS1-01 Yellow2,CS1-Onyx,REST Yellow3,CS1-Onyx,CS1-01,REST
FIELD2:Attacker,CS2-ShieldSmasher
HAND1:CS2-Rampager CS2-Colossal DW1-Van CS1-AdamantiumShield
GAME:P2 T3 MAIN 3
DUMP:HAND,DP

ECHO: -- attack into Yellow3, get dp boost, trigger blast
CMD:ATTACK Attacker Yellow3
EFFECT-TREE
GET-CHOICES
DUMP:HAND,DP


ECHO: -- choose first non-nothing
STEP:TO:1
EFFECT-TREE
GET-CHOICES
DUMP:HAND,DP

ECHO: -- the attacker was removed (oops, I didn't mean to test this). I shouldn't get the choice to block
