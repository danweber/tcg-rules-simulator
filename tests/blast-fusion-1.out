FIELD1:Blue,CS2-Animesword,CS2-Doggiedagger,12000DP Yellow1,CS1-Obsidianshield,CS1-Tinyshield,13000DP Yellow2,CS1-Onyxshield,REST,8000DP Yellow3,CS1-Onyxshield,CS1-Tinyshield,REST,8000DP
FIELD2:Attacker,CS2-Shieldsmasher,9000DP
HAND1:CS2-Rampager CS2-Colossalsword DW1-Vanillamon CS1-Adamantiumshield
HAND2:
GAME:P2 T3 MAIN 3
ECHO: -- attack into Yellow3, get dp boost, trigger blast
EFFECT-TREE:NOTHING
GET-CHOICES:Choose:
GET-CHOICES:Don't Blast Evolve
GET-CHOICES:Evolve Lv.7 Rampager onto Adamantium Shield in HAND and Anime Sword (0)
GET-CHOICES:Evolve Lv.7 Rampager onto Obsidian Shield and Colossal Sword in HAND (0)
GET-CHOICES:Evolve Lv.7 Rampager onto Onyx Shield and Colossal Sword in HAND (0)
GET-CHOICES:Evolve Lv.7 Rampager onto Onyx Shield and Colossal Sword in HAND (0)
FIELD1:Blue,CS2-Animesword,CS2-Doggiedagger,12000DP Yellow1,CS1-Obsidianshield,CS1-Tinyshield,13000DP Yellow2,CS1-Onyxshield,REST,8000DP Yellow3,CS1-Onyxshield,CS1-Tinyshield,REST,8000DP
FIELD2:Attacker,CS2-Shieldsmasher,REST,14000DP
HAND1:CS2-Rampager CS2-Colossalsword DW1-Vanillamon CS1-Adamantiumshield
HAND2:
GAME:P2 T3 MAIN 3
ECHO: -- choose first non-nothing
EFFECT-TREE:NOTHING
GET-CHOICES:Choose:
FIELD1:Yellow1,CS1-Obsidianshield,CS1-Tinyshield,13000DP Yellow2,CS1-Onyxshield,REST,8000DP Yellow3,CS1-Onyxshield,CS1-Tinyshield,REST,8000DP I6,CS2-Rampager,CS1-Adamantiumshield,CS2-Animesword,CS2-Doggiedagger,14000DP
FIELD2:
HAND1:CS2-Colossalsword DW1-Vanillamon
HAND2:
GAME:P2 T3 MAIN 3
ECHO: -- the attacker was removed (oops, I didn't mean to test this). I shouldn't get the choice to block
